maple/menu.c

```diff
@@ -414,67 +414,67 @@ static char footer[160];
 void
 movie(void)
 {
-  static int orig_flag = -1;
-  static time_t uptime = -1;
-  static char flagmsg[16];
-  static char datemsg[16];
+    static int orig_flag = -1;
+    static time_t uptime = -1;
+    static char flagmsg[16];
+    static char datemsg[16];
 
-  int ufo;
-  time_t now;
+    int ufo;
+    time_t now;
 
-  ufo = cuser.ufo;
-  time(&now);
+    ufo = cuser.ufo;
+    time(&now);
 
-  /* Thor: it is depend on which state */
+    /* Thor: it is depend on which state */
 
-  if ((bbsmode < M_CLASS) && (cuser.ufo2 & UFO2_MOVIE))
-  {
-    static int tag = FILM_MOVIE;
+    if ((bbsmode < M_CLASS) && (cuser.ufo2 & UFO2_MOVIE))
+    {
+        static int tag = FILM_MOVIE;
 
-    tag = film_out(tag, 2);
-  }
+        tag = film_out(tag, 2);
+    }
 
 
-  /* Thor: �P�� ��� �I�s�� �n�ͤW�� �� */
+    /* Thor: �P�� ��� �I�s�� �n�ͤW�� �� */
 
-  ufo &= UFO_PAGER | UFO_CLOAK | UFO_QUIET | UFO_PAGER1 | UFO_MESSAGE;
-  if (orig_flag != ufo)
-  {
-    orig_flag = ufo;
-    sprintf(flagmsg,
-      "%s/%s",
-      (ufo & UFO_PAGER1) ? "����" : (ufo & UFO_PAGER) ? "�b��" : "���}",
-      (ufo & UFO_MESSAGE) ? "����" : (ufo & UFO_QUIET) ? "�b��" : "���}");
-  }
+    ufo &= UFO_PAGER | UFO_CLOAK | UFO_QUIET | UFO_PAGER1 | UFO_MESSAGE;
+    if (orig_flag != ufo)
+    {
+        orig_flag = ufo;
+        sprintf(flagmsg,
+            "%s/%s",
+            (ufo & UFO_PAGER1) ? "����" : (ufo & UFO_PAGER) ? "�b��" : "���}",
+            (ufo & UFO_MESSAGE) ? "����" : (ufo & UFO_QUIET) ? "�b��" : "���}");
+    }
 
-  if (now > uptime)
-  {
-    struct tm *ptime;
+    if (now > uptime)
+    {
+        struct tm *ptime;
 
-    ptime = localtime(&now);
-    sprintf(datemsg, "[%d/%d �P��%.2s ",
-      ptime->tm_mon + 1, ptime->tm_mday,
-      "�Ѥ@�G�T�|����" + (ptime->tm_wday << 1));
+        ptime = localtime(&now);
+        sprintf(datemsg, "[%d/%d �P��%.2s ",
+            ptime->tm_mon + 1, ptime->tm_mday,
+            "�Ѥ@�G�T�|����" + (ptime->tm_wday << 1));
 
-    uptime = now + 86400 - ptime->tm_hour * 3600 -
-      ptime->tm_min * 60 - ptime->tm_sec;
-  }
-  ufo = (now - (uptime - 86400)) / 60;
+        uptime = now + 86400 - ptime->tm_hour * 3600 -
+            ptime->tm_min * 60 - ptime->tm_sec;
+    }
+    ufo = (now - (uptime - 86400)) / 60;
 
-  /* Thor.980913: ���: �̱`���I�s movie()���ɾ�O�C����s film, �b 60��H�W,
-                        �G���ݰw�� xx:yy �ӯS�O�@�@�r���x�s�H�[�t */
+    /* Thor.980913: ���: �̱`���I�s movie()���ɾ�O�C����s film, �b 60��H�W,
+                          �G���ݰw�� xx:yy �ӯS�O�@�@�r���x�s�H�[�t */
 
-  sprintf(footer, "\033[0;34;46m%s%d:%02d] \033[30;47m �ثe���W��\033[31m%4d\033[30m �H, �ڬO \033[31m%-12s\033[30m [�I�s/�T��]\033[31m%s",
-    datemsg, ufo / 60, ufo % 60,
-    /*ushm->count*/total_num, cuser.userid, flagmsg);
-  outf(footer);
+    sprintf(footer, "\033[0;34;46m%s%d:%02d] \033[30;47m �ثe���W��\033[31m%4d\033[30m �H�A�ڬO \033[31m%-12s\033[30m [�I�s/�T��]\033[31m%s\033[m",
```

Comment:
The reason I removed `\033[m` in 229a2d2 is that `outf()` function will automatically provides `\033[m` in the end of output message.
maybe we can add notes at that part later, or find better way to rewrite it.

Reply:
The `\033[m` was removed again in 229a2d2.